### PR TITLE
Intro to ADA QC edits

### PR DESCRIPTION
**Pull request by brentryanjohnson**
- Edits Intro to ADA based on quality control governance process

### CODE REVIEW

**Comment by hursey013** [comment]
After this is merged, merge https://github.com/usdoj-crt/beta-ada/pull/124

**Comment by brentryanjohnson** [comment]
Looks like @kstitch7 [has reviewed the edits](https://gsa-tts.slack.com/archives/C01543QESH0/p1622756045019200) 🔒, and we're good to go! Go ahead and merge if it looks good to you, @hursey013.

**Review by hursey013 — APPROVED** [review]

**Pull request merged**
